lib/configure.js

```diff
@@ -48,6 +48,7 @@ function configure (gyp, argv, callback) {
         }
       } else {
         log.verbose('`which` succeeded', python, execPath)
+        python = execPath
```

Comment:
Can you add a comment explaining that (Properly capitalized and punctuated, please.)?

It would be nice to have some kind of regression test for this.  Maybe you can `module.exports.test.checkPython = checkPython` and verify in the test that executing the result with `-V` prints the python version banner?